driver/src/order_driver.rs

```diff
@@ -83,11 +83,11 @@ fn compute_updated_balances(
     let mut result = balances.clone();
     for (index, order) in orders.iter().enumerate() {
```

Comment:
In case you might not be aware, I have a unit test for this function written in here:
https://github.com/gnosis/dex-services/pull/99/files#diff-37736b2e285edcecdc56bd9aacf3fceaL107

Since the old unit test was hopelessly broken